css/lemuel/styles.css

```diff
@@ -0,0 +1,19 @@
+* {
+    border-radius: 0 !important;
```

Comment:
I'm overriding Boostrap's default border radius values. 
It's just a lazy approach to removing the rounded corners. 

using the below values won't work, hence the reason for using important to force the override.

* {
        border-radius: 0;
}